### PR TITLE
Add ensure attribute for limits fragment

### PR DESCRIPTION
**Pull request by jwennerberg**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
@Phil-Friderici look good to you?

File: README.md [hunk 1]
```diff
@@ -406,6 +412,8 @@ list
 ----
 Array of lines to add to the fragment file
 
+- *Default*: undef
```
Comment (ghoneycutt) [inline README.md]: Good catch!

File: spec/defines/limits/fragment_spec.rb [hunk 2]
```diff
@@ -99,6 +99,49 @@
     }
   end
 
+  context 'with ensure set to absent' do
+    let(:title) { '80-nproc' }
+    let(:params) {
+      { :ensure => 'absent' }
```
Comment (Phil-Friderici) [inline spec/defines/limits/fragment_spec.rb]: please also test the other valid values 'present' and 'absent'

**Comment by Phil-Friderici** [comment]
functionality looks good to me :+1:

**Comment by jwennerberg** [comment]
travis fails because this is missing: https://github.com/jwennerberg/puppet-module-pam/commit/55abd50c8b84eb628660c9a07b2015f718a16eed

**Comment by ghoneycutt** [comment]
@jwennerberg Merged the fix. You should be good to go on doing the rebase against master and repushing.

**Comment by jwennerberg** [comment]
Closed in favor of #109

**Pull request closed without merging**